tutorials/onboarding/cloud_only.json

```diff
@@ -0,0 +1,170 @@
+    "ad_domain": "MyDomain.local",
+    "ad_admin": "hpcadmin",
+    "win_password": "<NOT-SET>"
+  },
+  "vnet": {
+    "name": "hpcvnet",
+    "address_prefix": "10.2.0.0/20",
+    "subnets": {
+      "infra": "10.2.0.0/24",
+      "compute": "10.2.4.0/22"
+    }
+  },
+  "resources": {
+    "adnode": {
+      "type": "vm",
+      "public_ip": "true",
+      "password": "variables.win_password",
+      "vm_type": "Standard_D8s_v3",
+      "image": "variables.ad_image",
+      "subnet": "infra",
+      "tags": [
+        "ad_server"
+      ]
+    },
+    "frontend": {
+      "type": "vm",
+      "public_ip": "true",
+      "password": "variables.win_password",
+      "vm_type": "Standard_D8s_v3",
+      "image": "variables.ad_image",
+      "subnet": "infra",
+      "tags": [
+      ]
+    },
+    "license": {
+      "type": "vm",
+      "vm_type": "Standard_D4s_v3",
+      "public_ip": true,
+      "image": "variables.image",
+      "subnet": "infra",
+      "tags": [
+ 	"cndefault",
+        "disable-selinux",
+	"join_domain"
+      ]
+    },
+    "headnode": {
+      "type": "vm",
+      "vm_type": "Standard_HB60rs",
+      "public_ip": true,
+      "image": "variables.image",
+      "subnet": "compute",
+      "tags": [
+ 	"cndefault",
+        "nfsserver",
+        "pbsserver",
+        "loginnode",
+        "localuser",
+        "disable-selinux",
+	"join_domain"
+      ]
+    },
+   "compute": {
+       "type": "vmss",
+      "vm_type": "variables.vm_type",
+      "instances": 8,
+      "image": "variables.hpc_image",
+      "subnet": "compute",
+      "tags": [
+        "nfsclient",
+        "pbsclient",
+        "cndefault",
+        "localuser",
+        "disable-selinux",
+        "join_domain"
+      ]
+    }
+  },
+  "install": [
+    {
+      "script": "setup_win_ad.sh",
+      "type": "local_script",
+      "args": [
+        "variables.resource_group",
+        "$(<hostlists/tags/ad_server)",
+        "variables.ad_domain",
+        "variables.ad_admin",
+        "variables.win_password"
+      ]
+    },
+    {
+      "script": "disable-selinux.sh",
+      "tag": "disable-selinux",
+      "sudo": true
+    },
+    {
+      "script": "cndefault.sh",
+      "tag": "cndefault",
+      "sudo": true
+    },
+    {
+      "script": "nfsserver.sh",
+      "tag": "nfsserver",
+      "sudo": true
+    },
+    {
+      "script": "nfsclient.sh",
+      "args": [
+        "$(<hostlists/tags/nfsserver)"
+      ],
+      "tag": "nfsclient",
+      "sudo": true
+    },
+    {
+      "script": "localuser.sh",
+      "args": [
+        "$(<hostlists/tags/nfsserver)"
+      ],
+      "tag": "localuser",
+      "sudo": true
+    },
+    {
+      "script": "pbsdownload.sh",
+      "tag": "loginnode",
+      "sudo": false
+    },
+    {
+      "script": "pbsserver.sh",
+      "copy": [
+        "pbspro_19.1.1.centos7/pbspro-server-19.1.1-0.x86_64.rpm"
+      ],
+      "tag": "pbsserver",
+      "sudo": false
```

Comment:
please change to true for all pbs scripts

Reply:
done